src/sentry/conf/server.py

```diff
@@ -1942,8 +1942,6 @@ def custom_parameter_sort(parameter: dict) -> tuple[str, int]:
     "organizations:user-feedback-ingest": False,
     # Use ReplayClipPreview inside the User Feedback Details panel
     "organizations:user-feedback-replay-clip": False,
-    # Enable User Feedback spam auto filtering feature UI
-    "organizations:user-feedback-spam-filter-ui": False,
```

Comment:
does spam filter work on self hosted?

Reply:
it should, @JoshFerge just told me to remove the flags😶‍🌫️

Reply:
i need to write the docs for it still, but for now self hosted can mark things manually as spam